src/main/drivers/rangefinder/rangefinder_virtual.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * This file is part of Betaflight and INAV
+ *
+ * Betaflight and INAV are free software. You can
+ * redistribute this software and/or modify this software under
+ * the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * Betaflight and INAV are distributed in the hope that
+ * they will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * This is a bridge driver between a sensor reader that operates at high level (i.e. on top of UART)
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#include "platform.h"
+
+#ifdef USE_RANGEFINDER_MSP
+#include "build/build_config.h"
+
+#include "common/utils.h"
+
+#include "drivers/rangefinder/rangefinder_virtual.h"
+
+static const virtualRangefinderVTable_t * highLevelDeviceVTable = NULL;
```

Comment:
`highLevelDevice` is strange name, and it's even more confusing when used with sensor returning height. In this file, deviceVTable may be enough.